m0t1fs/linux_kernel/st/m0t1fs_dgmode_io.sh

```diff
@@ -121,7 +121,7 @@ fmio_truncation_module()
 	seek=`expr $cnt \+ 1`
 	echo "Ensure that truncating to a size larger than the
 	      current one succeeds"
-	fmio_files_write dd bs=$blk_size count=1 seek=$seek
+	fmio_files_write dd bs=$blk_size count=1 seek="$seek"
```

Comment:
Why does the `bs=$blk_size` not need double quotes, while the `seek="$seek"` needs?

Reply:
@huanghua78 , Thanks for the review. I had installed shellcheck plugin in the IDE and whatever words were marked with red-color under underscore I tried fixing them all. 

However this is not all. We will be fixing more and more such issues in stages as there are too many such issues, 1688 occurrences to be precise. 

For now I will not fix the above as once that is done I will have to re-run the regression test again to prove that nothing else was broken because of the new change which will take 6+ hours to complete.

Also when this patch is merged we will not be closing issue-1606, as we are planning to post more and more patches in stages to fix all the occurrences of  this warning.